src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
+$width: 200px;
+
+* {
+  background: $white;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  background: #fff;
+}
+
+.card {
+  display: flex;
+  width: $width;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  .stars {
+    display: flex;
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background-image: url(../images/star.svg);
+      background-position: center ;
+      background-repeat: no-repeat;
+    }
+
+    &--4 .stars__star {
+      &:nth-child(-n+4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/ab4a4a5b-20c0-4871-a23c-3bd19dcce4bc)

add styles for --3, --2 etc